### DISCUSSION TITLE
How can I have this container utilize already downloaded Wikipedia and postcode resources?

### DISCUSSION THREAD

**Discussion by km6slf** [mediagis/nominatim-docker#282] [Q&A]
I already have wikimedia-importance.sql.gz and us_postcode_data.sql.gz files downloaded. I'm looking at the init.sh file, and I'm not sure how to switch it from downloading the file to reading it from local storage. Am I in the right place?

What do these sections need to be changed to in order to achieve this?
if [ "$IMPORT_WIKIPEDIA" = "true" ]; then
  echo "Downloading Wikipedia importance dump"
  curl https://nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
else
  echo "Skipping optional Wikipedia importance import"
fi;

f [ "$IMPORT_US_POSTCODES" = "true" ]; then
  curl https://nominatim.org/data/us_postcode_data.sql.gz -L -o ${PROJECT_DIR}/us_postcode_data.sql.gz
else
  echo "Skipping optional US postcode import"
fi;

**Comment by philipkozeny** [comment]
What we do here: check if an environment variable is set, to download the data into a specific directory. What you could do without changing anything: 

mount the two files into the appropriate locations (`${PROJECT_DIR}/wikimedia-importance.sql.gz`, `${PROJECT_DIR}/us_postcode_data.sql.gz`). They will be automatically used if they are present in that location.

So you would need to add something like ```-v /my/files/wikimedia-importance.sql.gz:/nominatim/wikimedia-importance.sql.gz``` depending on how you launch the image.

**Comment by km6slf** [comment]
Thanks!
